src/views/Login.vue

```diff
@@ -56,13 +56,11 @@
 							hide-details
 							label="Remember Me"
 						></v-checkbox>
-						<v-checkbox
+						<color-scheme
 							prepend-icon="null"
 							hide-details
-							persistent-hint
-							label="Dark mode"
-							v-model="internalDarkMode"
-						></v-checkbox>
+							v-model="internalColorScheme"
+						></color-scheme>
```

Comment:
there is not need to specify v-model here, the logic in this case will be handled inside the component as it works with values from the store :)